### PR TITLE
Fix maintainers list into a table for easy read.

### PR DESCRIPTION
**Pull request by KojoShaddy**

### CODE REVIEW

**Comment by KojoShaddy** [comment]
> @googlebot I signed it!

File: research/object_detection/README.md [hunk 1]
```diff
@@ -27,14 +27,17 @@ https://scholar.googleusercontent.com/scholar.bib?q=info:l291WsrB-hQJ:scholar.go
...
-* Ronny Votel, github: [ronnyvotel](https://github.com/ronnyvotel)
-* Derek Chow, github: [derekjchow](https://github.com/derekjchow)
-* Chen Sun, github: [jesu9](https://github.com/jesu9)
-* Menglong Zhu, github: [dreamdragon](https://github.com/dreamdragon)
-* Alireza Fathi, github: [afathi3](https://github.com/afathi3)
-* Zhichao Lu, github: [pkulzc](https://github.com/pkulzc)
+| Name | GitHub |
+| --- | --- |
+| Jonathan Huang | [jch1](https://github.com/jch1) | 
+| Jonathan Huang | [jch1](https://github.com/jch1) |
```
Comment (lc0) [inline research/object_detection/README.md:33]:
```suggestion
| Jonathan Huang | [jch1](https://github.com/jch1) | 
```

Seems like you duplicated first row

Comment (KojoShaddy) [reply research/object_detection/README.md]: Resolved. Thanks for alert.

**Comment by KojoShaddy** [comment]
Thanks for merging my pull request.

**Pull request merged**